### ISSUE TITLE
docker-compose for test suite

### ISSUE THREAD

**Issue by 3nprob** [bitromortac/lndmanage#122]
Would be sweet if there was a docker-compose that wired up the environment necessary to run the integration tests

**Comment by bitromortac** [comment]
Yes, that would be cool. I should also have a look into CI at some point.
